homework/H2_Linear_Algebra.jl

```diff
@@ -756,16 +755,13 @@ $H = W_k^T \cdot \hat{X}$
 # ╔═╡ 216dcbf9-91aa-40fe-bef2-6dea56a35be7
 function pca(X; k=3)  # Don't change this line
     # Write your code here
-	
-    W_k = missing # Replace `missing` with your code
-    H = missing # Replace `missing` with your code
     
     return W_k, H # Don't change this line
 end
 
 # ╔═╡ dda486b4-5838-429b-aec8-450d2f0c55be
 hint(md"
-* the `LinearAlgebra` packagage offers valueable functions such as `mean` and `eigen`
+* the `LinearAlgebra` and `Statistics` packagages offer valueable functions such as `mean` and `eigen` (Note: you might need to import Statistics).
```

Comment:
Ideally we should be the ones importing all relevant packages to make the notebooks reproducible.
This reproducibility issue isn't super relevant here because Statistics.jl is a Base package, but I think it is good practice and makes things easier for students.

Reply:
added both `Statistics`  and `Distributions` at the top of the file. The latter is used for calculating a random gaussian sample for the visualisation.  This was done in [adding Visualisation to PCA, HW2](https://github.com/adrhill/julia-ml-course/pull/4/commits/c91afe766a212a9127a9e8fe9cb0d29d7e8f94b5)